<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>showClass</class>
 <widget class="QWidget" name="showClass">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>692</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>300</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>561</width>
     <height>222</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QComboBox" name="comboBox">
        <item>
         <property name="text">
          <string>大一上学期</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>大一下学期</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>大二上学期</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>大二下学期</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>大三上学期</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>大三下学期</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>大四上学期</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>大四下学期</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>年</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_2"/>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>月</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_3"/>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>日</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QListView" name="listView"/>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
